types/dom-chromium-ai/dom-chromium-ai-tests.ts

```diff
@@ -1,134 +1,175 @@
 async function topLevel() {
     // Language Model
 
-    // Negative tests
-    await self.ai.languageModel.create({
-        systemPrompt: "foo",
-        // @ts-expect-error - System prompt cannot be part of the array if systemPrompt is specified.
-        initialPrompts: [{ role: "system" }],
-    });
-    await self.ai.languageModel.create({
+    await LanguageModel.create({
         // @ts-expect-error - System prompt must be first element of the initialPrompt array.
-        initialPrompts: [{ role: "user" }, { role: "system" }],
+        initialPrompts: [{ role: "user", content: "foo" }, { role: "system", content: "foo" }],
     });
 
-    // Positive tests
-    // System prompt in create options
-    const languageModel1 = await self.ai.languageModel.create({
+    const languageModel = await LanguageModel.create({
         topK: 1,
         temperature: 0,
+        expectedInputs: [{ type: "text", languages: ["de"] }],
+        expectedOutputs: [{ type: "image", languages: ["de"] }],
```

Comment:
```suggestion
        expectedOutputs: [{ type: "text", languages: ["de"] }],
```

This <a href="https://github.com/webmachinelearning/prompt-api#:~:text=(Note%20that%20presently%2C%20the%20prompt%20API%20does%20not%20support%20multimodal%20outputs%2C%20so%20including%20anything%20array%20entries%20with%20types%20other%20than%20%22text%22%20will%20always%20fail.">isn't currently supported</a>.